packages/aws-cdk-lib/aws-cognito/test/user-pool.test.ts

```diff
@@ -161,14 +161,35 @@ describe('User Pool', () => {
         emailStyle: VerificationEmailStyle.LINK,
         emailBody: 'invalid email body {####}',
       },
-    })).toThrow(/Verification email body/);
+    })).not.toThrow();
 
     expect(() => new UserPool(stack, 'Pool6', {
       userVerification: {
         emailStyle: VerificationEmailStyle.LINK,
         emailBody: 'invalid email body {##Verify Email##}',
       },
     })).not.toThrow();
+
+    expect(() => new UserPool(stack, 'Pool7', {
+      userVerification: {
+        emailStyle: VerificationEmailStyle.LINK,
+        emailBody: 'invalid email body ##Verify Email##',
+      },
+    })).toThrow(/Verification email body/);
+
+    expect(() => new UserPool(stack, 'Pool8', {
+      userVerification: {
+        emailStyle: VerificationEmailStyle.LINK,
+        emailBody: 'invalid email body {##Verify !! Email##}',
+      },
+    })).toThrow(/Verification email body/);
+
+    expect(() => new UserPool(stack, 'Pool9', {
+      userVerification: {
+        emailStyle: VerificationEmailStyle.LINK,
+        emailBody: 'invalid email body {##Click here to verify##}',
```

Comment:
this `emailBody` is valid right? if so it's a bit confusing that it says 'invalid email body'

Reply:
good catch.